Client/src/pages/pantry.vue

```diff
@@ -28,14 +29,17 @@
     <q-dialog
       v-model="clickedCall"
     >
-      <q-card style="width: 300px">
+      <q-card style="width: 300px" >
         <div class="q-pd-md">
           <q-card-section>
-            Email: {{ selectedOrganization.email }}
+
           </q-card-section>
 
-          <q-card-section>
-            Phone: {{ selectedOrganization.phone }}
+          <q-card-section v-for="(user, index) in selectedOrganization" :key="index">
```

Comment:
@Daniel-Eprog Do we need the v-for? Are we displaying the email and phone for multiple organizations in one card? If not, then can remove the v-for and put email and phone in separate q-sections.